import.sh

```diff
@@ -0,0 +1,46 @@
+#!/bin/bash
+
+#json and jsonnet files
+mkdir govuk_content_schemas
+cp -r ../govuk-content-schemas/formats ./govuk_content_schemas/formats
+cp -r ../govuk-content-schemas/dist ./govuk_content_schemas/dist
```

Comment:
```suggestion
mkdir content_schemas
cp -r ../govuk-content-schemas/formats ./content_schemas/formats
cp -r ../govuk-content-schemas/dist ./content_schemas/dist
```

I think we can remove the formality of the `govuk`. I think it risks things being a little confused as we don't call this the GOV.UK Publishing API so it'd be simpler with them as just content schemas.

I also wonder if the directory would feel idiomatically placed under config, e.g `config/content_schemas` since this is configuration. Not strongly opined on that though.